app/views/orders/_form.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1 Select your meals for today
+= form_for [@facade.menu, @facade.new_order] do |f|
+  = render @facade.meals.select{ |m| m.course == "First Course"}
+  = render @facade.meals.select{ |m| m.course == "Main Course"}
+  = render @facade.meals.select{ |m| m.course == "Beverage"}
```

Comment:
This logic should not reside in the view. 
Also, think of a better way to tackle this problem.